frontends/web/src/routes/device/bitbox02/checkbackup.tsx

```diff
@@ -46,66 +46,70 @@ class Check extends Component<Props, State> {
-        alertUser(this.props.t('unknownError', { errorMessage: 'Not found' }));
+        alertUser(t('unknownError', { errorMessage: 'Not found' }));
         return;
       }
       this.setState({
-        foundBackup,
         activeDialog: true,
-        message: this.props.t('backup.check.success'),
+        foundBackup,
+      });
+      await bitbox02API.checkBackup(deviceID, false);
+      this.setState({
+        message: t('backup.check.success'),
+        userVerified: true,
       });
-      await bitbox02API.checkBackup(this.props.deviceID, false);
-      this.setState({ userVerified: true });
     } catch {
-      this.setState({ activeDialog: true, message: this.props.t('backup.check.notOK'), userVerified: true });
+      this.setState({
+        activeDialog: true,
+        message: t('backup.check.notOK'),
+        userVerified: true,
+      });
     }
   };
 
-  private abort = () => {
-    this.setState({ activeDialog: false, userVerified: false });
-  };
-
   public render() {
-    const { t } = this.props;
+    const { disabled, t } = this.props;
     const { activeDialog, message, foundBackup, userVerified } = this.state;
     return (
       <div>
         <Button
           primary
-          disabled={this.props.disabled}
+          disabled={disabled}
           onClick={this.checkBackup}
         >
           {t('button.check')}
         </Button>
         <Dialog open={activeDialog} title={message}>
-          <div className="columnsContainer half">
-            <div className="columns">
-              <div className="column">
-                {
-                  foundBackup !== undefined && (
-                    <BackupsListItem
-                      backup={foundBackup}
-                      handleChange={() => undefined}
-                      onFocus={() => undefined}
-                      radio={false} />
-                  )
-                }
-              </div>
-            </div>
+          <form onSubmit={(e) => {
+            e.preventDefault();
+            this.setState({
+              activeDialog: false,
+              userVerified: false,
+            });
+          }}>
+            { foundBackup !== undefined && (
+              <BackupsListItem
+                backup={foundBackup}
+                radio={false} />
+            )}
             <DialogButtons>
-              <Button
-                primary
-                onClick={this.abort}
-                disabled={!userVerified}
-              >
-                { userVerified ? t('button.ok') : t('accountInfo.verify') }
-              </Button>
+              {userVerified && (
+                <Button
+                  autoFocus
+                  disabled={!userVerified}
+                  primary
+                  type="submit">
+                  { userVerified ? t('button.ok') : t('accountInfo.verify') }
+                </Button>
+              )}
             </DialogButtons>
-          </div>
+          </form>
```

Comment:
changing to a form that has onSubmit event listener, and button type submit with autofocus, allows for confirming the dialog with pressing "Enter"